forge-gui/res/cardsfolder/upcoming/silent_extraction.txt

```diff
@@ -0,0 +1,9 @@
+Name:Silent Extraction
+ManaCost:G U
+Types:Instant
+A:SP$ Pump | TargetType$ Spell | TgtZone$ Stack | ValidTgts$ Card | SubAbility$ DBCounter | SpellDescription$ Choose target spell.
```

Comment:
why extra Pump?

Reply:
I think it's need to allow you to seek if the card isn't countered ?

Reply:
doesn't make sense
Targeted stays the same either way